core-chart/templates/postcreatehooks/install-status-addon.yaml

```diff
@@ -41,38 +41,30 @@ spec:
               - sh
               - -c
               - |
-                echo "Waiting for OCM CRDs to be established..."
-                # Wait for essential OCM CRDs needed by the Status Add-on
-                # First check if clusteradm has created any OCM CRDs
-                echo "Checking if OCM CRDs exist..."
+                echo "Waiting for essential OCM CRDs to become available..."
                 max_attempts=60
                 attempt=0
                 while [ $attempt -lt $max_attempts ]; do
-                  if kubectl get crd placements.cluster.open-cluster-management.io >/dev/null 2>&1; then
-                    echo "OCM CRDs found, waiting for them to be established..."
-                    kubectl wait --for=condition=Established crd/placements.cluster.open-cluster-management.io --timeout=300s
```

Comment:
The wait for `condition=Established` is critical. It is not enough for the CRD object to exist. A CRD object is an instruction to the kube-apiservers to add a "resource" (kind of API object). They do this asynchronously after the CRD object is created. The condition "Established" signals completion of this critical asynchronous work.